src.ts/utils/key_pair.ts

```diff
@@ -86,6 +86,20 @@ export abstract class KeyPair {
             throw new Error('Invalid encoded key format, must be <curve>:<encoded key>');
         }
     }
+    
+    static fromSeed(seedStr: string): KeyPair {
```

Comment:
What is the purpose for this, do we need to support this specific format of seed phrases?

Note that we already have library with support for conforming BIP39 seed phrases (compatible with other wallets):
https://github.com/nearprotocol/near-seed-phrase/blob/master/index.js

Reply:
Good point! seems this change isn't very helpful for a general api-js user and they should seek for near-seed-phrase